applications/StructuralMechanicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -70,6 +71,17 @@ void  AddCustomUtilitiesToPython()
     .def("Transfer",TransferEigenvector2)
     ;
 
+    class_<MpcData, boost::noncopyable>( "MpcData", init<>() )
+	.def("AddSlave",&MpcData::AddSlave)
```

Comment:
when you add a slave, you will always have also to assign a master to it. so why don't you do it in a single function?
this will avoid mistakes